bika/lims/validators.py

```diff
@@ -93,7 +92,7 @@ def __call__(self, value, *args, **kwargs):
         if value == instance.get(fieldname):
             return True
 
-        for item in aq_parent(instance).objectValues():
+        for item in instance.aq_parent.objectValues():
```

Comment:
This modif does not belong to this Pull Request. In any case, better to replace the implementation of `UniqueFieldValidator` by this one:

https://github.com/bikalims/bika.lims/blob/master/bika/lims/validators.py#L79-L146

Is more elaborated and less expensive. @Nihadness , do you mind to create a new PR, different with the replacement mentioned?